drivers/cpuquiet/governors/rqbalance.c

```diff
@@ -555,13 +555,18 @@ static void rqbalance_work_func(struct work_struct *work)
 		break;
 	case DOWN:
 		cpu = get_slowest_cpu_n();
-		if (cpu < nr_cpu_ids)
+		if (cpu < nr_cpu_ids) {
 			up = false;
+			// Next time recheck if CPU is balanced
+			rqbalance_state = UP;
+		}
 		else
 			stop_load_timer();
 
-			queue_delayed_work(rqbalance_wq,
-						 &rqbalance_work, up_delay);
+
+
+		queue_delayed_work(rqbalance_wq,
```

Comment:
@djselbeck can you update this ..... string 558 - 562 are the commit string 566-658 is useless

Reply:
I agree about the useless whitelines

Reply:
Yes the lines should be fixed, sadly I noticed them to late. But the indentation should be fixed I guess?

Reply:
I don't see any indentation problem

Reply:
you should rebase ... this is useless commit